### DISCUSSION TITLE
Configutation to start develop a SPA

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [alephjs/aleph.js#216] [Q&A] [answered]
From the documentation, there is a flag to start the develop the application in SPA mode, but in the examples this flag it is not present.

What is the current configuration to enable the SPA mode?

**Comment by shadowtime2000** [comment, marked as answer]
[check this](https://github.com/alephjs/aleph.js/blob/master/examples/hello-world-spa/aleph.config.ts#L5)

**Reply by vincenzopalazzo** [reply]
added after I ask with this commit https://github.com/alephjs/aleph.js/commit/2ae45401edb3651701824f7ecfb340886195c128#diff-e48de03c5614279c63e8062b8306ee9368c8971df8470c038de16f50276a3e87, so this was not a stupid question :-)

**Comment by aadamsx** [comment]
What does SPA mode mean in this context, client side render?  If that's what you mean, then how does Aleph work in this case, does it push down the entire codebase on first request like SPAs and from that point we make async client side requests for data?
